mw-webapp/src/logic/allUsersPage/AllUsersPage.tsx

```diff
@@ -45,8 +73,26 @@ export const AllUsersPage = () => {
   }
 
   return (
-    <ScrollableBlock>
-      <UsersTableBlock users={allUsers} />
-    </ScrollableBlock>
+    <>
```

Comment:
extra wrapper

Reply:
It;s not extra wrapper. Inside few children